spacepy/lib.py

```diff
@@ -119,11 +121,15 @@ def load_lib():
 def load_call_dict(call_dict, lib):
     """Loads argument/return types from the call dictionary
 
-    @param call_dict: call dictionary. Keyed by function name;
-                      values are [return type, argtype0, argtype 1...]
-    @type call_dict: dict
-    @param lib: library where functions specified in L{call_dict} live.
-    @type lib: ctypes.WinDLL or ctypes.CDLL
+    Parameters
+    ==========
+    call_dict : dict
+        call dictionary. Keyed by function name;
+        values are [return type, argtype0, argtype 1...]
+    
+    lib : ctypes.WinDLL or ctypes.CDLL
+        library where functions specified in L{call_dict} live.
```

Comment:
Just CDLL. Also Sphinx doesn't support argument linking (epydoc's one advantage) so this should just be:
```
``call_dict``
```
...we just render parameter references as code

Reply:
resolved